lib/msal-core/tests/MSAL.spec.ts

```diff
@@ -166,12 +167,12 @@ describe('Msal', function (): any {
         global.document = documentMock;
         global.Math = mathMock;
 
-        msal = new UserAgentApplication("0813e1d1-ad72-46a9-8665-399bba48c201", null, function (errorDes, token, error) {
-        });
-        msal._user = null;
-        msal._renewStates = [];
-        msal._activeRenewals = {};
-        msal._cacheStorage = storageFake;
+        let config = buildConfiguration({clientId: "0813e1d1-ad72-46a9-8665-399bba48c201" }, {}, {}, {});
```

Comment:
is there a way to not have to pass these {} objects?

Reply:
This is in my pending AI, I will write some sample code and confirm this next week.

Reply:
Resolving for merge, we can update with a separate PR next week.